composer.json

```diff
@@ -12,7 +12,7 @@
 		],
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": ">=7.2",
```

Comment:
I'd recommend rather `^7.2|^8.0` so this release of the component won't be usable with future major version of php that could be BC break